components/raftstore/src/store/worker/read.rs

```diff
@@ -573,7 +600,14 @@ where
                             self.redirect(RaftCommand::new(req, cb));
                             return;
                         }
-                        self.execute(&req, &delegate.region, None, read_id)
+                        let response = self.execute(&req, &delegate.region, None, read_id);
+                        // Try renew lease in advance
+                        delegate.maybe_renew_lease_advance(
```

Comment:
Any benchmark to show the impact?

Reply:
What it cost is only two atomic operations, checking the `expired_time` and the `renewing` flag. The `expired_time` op can be further eliminated by loading once and reuse for both lease checking and renew lease checking, but it makes the code complicated though.

Reply:
Read itself is also a lightweight operation. I think point get benchmark should show the difference if any.